extensions/wikia/CommunityPage/templates/allAdmins.mustache

```diff
@@ -1,7 +1,7 @@
 <h2>{{allMembersHeaderText}}</h2>
 <ul class="reset top-contributors">
 	{{#admins}}
-		<li>
+		<li class="contributor">
```

Comment:
I wanted to refactor whole CommunityPage ext in next iteration to follow BEM convention but since we don't have that in guidelines for app this path is probably just fine. I'll add comm-page prefix.